app/config.js

```diff
@@ -163,23 +142,13 @@ export function getDcrdPath() {
   }
 }
 
-export function getCert() {
-  var certPath = "";
-  var cfg = getCfg();
-
-  if (cfg.get("cert_path") != "") {
-    certPath = cfg.get("cert_path");
-  }
-  if (os.platform() == "win32") {
-    certPath = path.join(os.homedir(), "AppData", "Local", "Decrediton", "rpc.cert");
-  } else if (os.platform() == "darwin") {
-    certPath = path.join(os.homedir(), "Library", "Application Support",
-            "decrediton", "rpc.cert");
-  } else {
-    certPath = path.join(os.homedir(), ".config", "decrediton", "rpc.cert");
-  }
+export function getWalletPath(testnet, walletPath) {
+  return path.join(appDataDirectory(), "wallets", testnet ? "testnet" : "mainnet", walletPath);
```

Comment:
On a second review, this seems to depend. When called from the main node process, it gets a boolean, but when called from the react app it gets a string. Probably passing the `network` selector instead of `isTestnet` somewhere.